fendermint/vm/genesis/src/lib.rs

```diff
@@ -17,6 +18,9 @@ use fendermint_vm_encoding::IsHumanReadable;
 #[cfg(feature = "arb")]
 mod arb;
 
+/// Power conversion decimal points, e.g. 3 decimals means 1 power per milliFIL.
+pub type PowerScale = i32;
```

Comment:
I am using `uint8` for `power_scale` in the contract (I guess it shouldn't be an issue).

Reply:
I allow negative values because -1 makes perfect sense depending on the value of your token: in our case you'd get a power for every 10 FIL. It would be great not to have an exception trying to pass it in.

Reply:
Gotcha, fixed in the contract.